docs/sources/plugins/developing/plugin-review-guidelines.md

```diff
@@ -57,7 +57,7 @@ A minimal `plugin.json` file:
 }
 ```
 
-- The convention for the plugin id is [github username/org]-[plugin name]-[datasource|app|panel] and it has to be unique. Although if org and plugin name are the same then [plugin name]-[datasource|app|panel] is also valid. The org **cannot** be `grafana` unless it is a plugin created by the Grafana core team.
+- The convention for the plugin id is **[github username/org]-[plugin name]-[datasource|app|panel]** and it has to be unique. Although if org and plugin name are the same then **[plugin name]-[datasource|app|panel]** is also valid. The org **cannot** be `grafana` unless it is a plugin created by the Grafana core team.
```

Comment:
internally we use `[plugin name]-[datasource|app|panel]` but for external plugins, it is important to use:

`[grafana.com username/org]-[plugin name]-[datasource|app|panel]`

Note I changed it to grafana.com, not github because that is what we really check